sdk/ml/azure-ai-ml/azure/ai/ml/entities/_job/job.py

```diff
@@ -318,7 +318,7 @@ def _from_rest_object(cls, obj: Union[JobBase, Run]) -> "Job":  # pylint: disabl
                 message=str(ex),
                 no_personal_data_message=f"Unable to parse a job resource of type:{type(obj).__name__}",
                 error_category=ErrorCategory.SYSTEM_ERROR,
-            )
+            ) from ex
```

Comment:
Should this be 'from e' instead?

Reply:
No, the tool I used tries to minimize the diff by detecting and re-using an existing exception name.

That exception was named `ex`:

https://github.com/Azure/azure-sdk-for-python/blob/e0a67a777ce69f1df5b0f2a8a5cbc8b7ef59acc6/sdk/ml/azure-ai-ml/azure/ai/ml/entities/_job/job.py#L312-L321